tasks/pipeline.py

```diff
@@ -418,9 +418,8 @@ def trigger_child_pipeline(_, git_ref, project_name, variable=None, follow=True)
 
     repo = get_gitlab_repo(project_name)
 
-    data = {"token": os.environ['CI_JOB_TOKEN'], "ref": git_ref, "variables": {}}
-
     # Fill the environment variables to pass to the child pipeline.
+    variables = {}
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
I would rename `variable` to `variables` or `variable_names` and add a type, it is not straightforward that `variable` is iterable and contains multiple variables. Then would rename current `variables` to `env_variables` or `child_pipeline_variables`